belgif-rest-problem/src/test/java/io/github/belgif/rest/problem/validation/RangeValidatorTest.java

```diff
@@ -0,0 +1,93 @@
+package io.github.belgif.rest.problem.validation;
+
+import static io.github.belgif.rest.problem.api.InEnum.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import io.github.belgif.rest.problem.api.Input;
+import io.github.belgif.rest.problem.api.InputValidationIssue;
+import io.github.belgif.rest.problem.api.InputValidationIssues;
+
+class RangeValidatorTest {
+
+    private RangeValidator<Integer> tested;
+
+    @Test
+    void validateOk() {
+        tested = new RangeValidator<>(Input.query("page", 3), 0, 5);
+        assertThat(tested.validate()).isEmpty();
+        tested = new RangeValidator<>(Input.query("page", 5), 0, 5);
+        assertThat(tested.validate()).isEmpty();
+        tested = new RangeValidator<>(Input.query("page", 0), 0, 5);
+        assertThat(tested.validate()).isEmpty();
+        tested = new RangeValidator<>(Input.query("page", 3), 0, null);
+        assertThat(tested.validate()).isEmpty();
+        tested = new RangeValidator<>(Input.query("page", 0), 0, null);
+        assertThat(tested.validate()).isEmpty();
+        tested = new RangeValidator<>(Input.query("page", 3), null, 5);
+        assertThat(tested.validate()).isEmpty();
+        tested = new RangeValidator<>(Input.query("page", 5), null, 5);
+        assertThat(tested.validate()).isEmpty();
+    }
+
+    @Test
+    void validateNullMinAndMax() {
+        assertThrows(IllegalArgumentException.class, () -> new RangeValidator<>(Input.query("page", 3), null, null),
```

Comment:
I would also use AssertJ for the exception assertions:

```
        assertThatExceptionOfType(IllegalArgumentException.class)
                .isThrownBy(() -> new RangeValidator<>(Input.query("page", 3), null, null))
                .withMessage("At least one of min, max must be non-null");
```

Reply:
Yes, I will do that in another general PR that improves the tests.